### ISSUE TITLE
Video loses alternative navigation interactivity after it starts; the user was unable to pause or interact with it.

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/notification-planning#2110]
Investigate into html video support with AT

https://app.makeitfable.com/share/685ec970f3aa3d71c7334a1a?video=1

Issue identified on:
Jaws for Windows 2025
Windows 11
Edge

**Comment by jzbahrai** [comment]
We can't fix this due to Jaws not rendering html5 correctly, would like to close

**Comment by connorscarolyns** [comment]
@YedidaZalik, we should add this to the known issues page and close this issue

**Comment by YedidaZalik** [comment]
@connorscarolyns could we add for the next iteration as we're almost finished this iteration?

**Comment by connorscarolyns** [comment]
yes, this can wait.
